src/app/page.tsx

```diff
@@ -36,7 +37,12 @@ export default function Home() {
           currentSong={currentSong}
         />
       </main>
-      <footer>footer</footer>
+
+      <section className="lyrics-section">
+        <h2 className="text-xl font-bold">Lyrics (in Polish)</h2>
```

Comment:
heading(h2) and section are parts of lyrics component so they should be inside it